src/view/com/auth/SplashScreen.tsx

```diff
@@ -29,63 +20,40 @@ export const SplashScreen = ({
 
   return (
     <CenteredView style={[styles.container, pal.view]}>
-      {onDismiss && (
-        <Pressable
-          accessibilityRole="button"
-          style={{
-            position: 'absolute',
-            top: 20,
-            right: 20,
-            padding: 20,
-            zIndex: 100,
-          }}
-          onPress={onDismiss}>
-          <FontAwesomeIcon
-            icon="x"
-            size={24}
-            style={{
-              color: String(pal.text.color),
-            }}
-          />
-        </Pressable>
-      )}
-
-      <SafeAreaView testID="noSessionView" style={styles.container}>
-        <ErrorBoundary>
-          <View style={styles.hero}>
-            <Text style={[styles.title, pal.link]}>
-              <Trans>Bluesky</Trans>
+      <ErrorBoundary>
+        <View style={styles.hero}>
```

Comment:
Moved the dismiss button to LoggedOut and removed SafeAreaView since this whole thing is already wrapped in SafeAreaView.